### DISCUSSION TITLE
multiple lines on a monitor and displaying different messages

### DISCUSSION THREAD

**Discussion by Experiment668** [cc-tweaked/CC-Tweaked#1914] [Q&A] [answered]
i have a 2H x 5L advanced monitor and i was hoping for some adives on how to dislpay text on multiple lines and how to cahnge a line of text between messages 
![Screenshot 2024-07-28 172958](https://github.com/user-attachments/assets/1df8a3e8-ed78-42b2-9bec-232318871dbc)
![Screenshot 2024-07-28 172945](https://github.com/user-attachments/assets/a44e4f78-a588-4765-939d-88dec92cf5a0)

**Comment by Experiment668** [comment]
![image](https://github.com/user-attachments/assets/81f84fb1-c1ee-489d-ab68-0f84156318a9)
got the 2nd line working not sure how to scale it down to be smaller

**Reply by SquidDev** [reply, marked as answer]
Do you want to scale down all the text (in which case you can just reduce the argument to `m.setTextScale` or just the bottom line?

The latter of these is much harder, as CC only supports a single font size for the whole monitor (much like a computer terminal). It's technically possible to emulate this with something like [bigfont](https://pastebin.com/3LfWxRWh), which draws larger text using CC's special "drawing characters", but I'm not sure I'd recommend it if you're just starting out with CC.
